x-pack/plugins/fleet/server/services/agent_policies/full_agent_policy.ts

```diff
@@ -491,13 +490,8 @@ export function transformOutputToFullPolicyOutput(
 
 /**
  * Get id used in full agent policy (sent to the agents)
- * we use "default" for the default policy to avoid breaking changes
  */
 function getOutputIdForAgentPolicy(output: Output) {
-  if (output.is_default) {
```

Comment:
removed this logic, it was added in 8.0: https://github.com/elastic/kibana/pull/111002